src/lib/replication/repl_dev/raft_state_machine.cpp

```diff
@@ -222,8 +222,6 @@ void RaftStateMachine::unlink_lsn_to_req(int64_t lsn) {
 void RaftStateMachine::link_lsn_to_req(repl_req_ptr_t rreq, int64_t lsn) {
     rreq->set_lsn(lsn);
     rreq->add_state(repl_req_state_t::LOG_RECEIVED);
-    // reset the rreq created_at time to now https://github.com/eBay/HomeStore/issues/506
-    rreq->set_created_time();
```

Comment:
is this still needed?

Reply:
no as I changed the `is_expired()` implementation.